package.json

```diff
@@ -111,9 +111,13 @@
     "yargs": "6.6.0"
   },
   "dependencies": {
+    "angular-bootstrap-switch": "^0.5.1",
+    "angular-dragdrop": "^1.0.13",
+    "angular-ui-sortable": "^0.16.1",
     "express": "4.14.0",
     "http-proxy": "1.16.2",
-    "manageiq-ui-components": "0.0.11",
+    "jquery-ui-bundle": "^1.11.4",
+    "manageiq-ui-components": "romanblanco/ui-components#dialog_editor",
```

Comment:
Thanks for the explanation @karelhala! I'm wondering if there is a possibility for `ui-components` to export individual components in addition to the entire library. That way dependencies can be contained within the library and consumers can import only those that they need. I have seen examples of this in libraries like `lodash` and `rxjs` and in some react component libraries.